### PR TITLE
[4.0] Atum font sizes

### PR DESCRIPTION
**Pull request by brianteeman**
PR for #31184

I took a look at this to create a PR the correct way for bootstrap scss

We should just have to change one variable
`$font-size-base:             	    0.875rem; //instead of 1rem`

But the rest of the font variables have not been defined correctly (probably based on bs4 beta1?) and they should be changed as well to be relative to this variable.

See the commit to bootstrap 3 years ago where this was done https://github.com/twbs/bootstrap/pull/24060

There may be other font sizes in our scss that need to be updated to use this variable - ~I have not checked yet~ _i believe I have done them all now but I deliberately ignored anything to do with icon fonts_

_**I have no opinion on a base font size of 16px (before) and 14px (with this PR)**_

To test you will need to run `npm ci` or `node build.js --compile-css`

### CODE REVIEW

File: administrator/templates/atum/scss/_variables.scss [hunk 1]
```diff
@@ -110,21 +110,22 @@ $sidebar-width-closed:             3rem;
 $main-brand-height:                3rem;
 
 // Fonts
+$font-size-base:             	     0.875rem; // Instead of default 1rem
```
Comment (infograf768) [inline administrator/templates/atum/scss/_variables.scss]: CS => change to `.875rem`

**Comment by richard67** [comment]
@brianteeman Could you update the branch for this PR with latest 4.0-dev of the CMS? Currently scss-cs is failing in Drone, but not due to code style of your PR. It seems the scss linter can't be started due to recently merged changes in the 4.0-dev branch of the CMS. In case if your 4.0-dev branch was not up to date with this when you have created the branch for this PR, this might be the reason. Thanks in advance.

**Comment by bembelimen** [comment]
The changes looks good, is there any a11y requirement based on font size? I know, that is should be possible to resize up to 200% without broken layout, but is there any min size?

PS: did you add other PRs into your PR? The JS stuff looks irrelevant?

**Comment by brianteeman** [comment]
grrh I updated the branch as requested and it looks like I did something wrong at the same time.

Regarding accessibility there are no rules or even recommendations about font sizes either minimum or maximum. The rules that applies are regarding contrast and a recommendation to use rem units to help scale content with a user’s personal preferences.

**Comment by brianteeman** [comment]
@bembelimen @richard67 the unrelated changes are coming from #31158 but they are already merged so I dont understand why they are showing up here

**Comment by richard67** [comment]
No idea either why they show up.

**Comment by brianteeman** [comment]
Closing and will create a new pr with a new branch

**Comment by brianteeman** [comment]
see #31207

**Pull request closed without merging**